Sources/NIOHTTP2/HTTP2FrameParser.swift

```diff
@@ -146,14 +177,20 @@ struct HTTP2FrameDecoder {
     private struct AwaitingPaddingLengthByteParserState {
         private var header: FrameHeader
         private var accumulatedBytes: ByteBuffer
+        private var maximumSequentialContinuationFrames: Int
 
-        init(fromAccumulatingFrameHeader state: AccumulatingFrameHeaderParserState, frameHeader: FrameHeader) {
```

Comment:
The `AccumulatingFrameHeaderParserState` stores `maximumSequentialContinuationFrames` so we don't need to pass it in again, we can just copy it from `state`.